docs/source/settingup.rst

```diff
@@ -151,6 +151,42 @@ Once the dataset is available on Dataverse, it can also be cloned using the
       'datalad-annex::?type=external&externaltype=dataverse&encryption=none&exporttree=no&url=https%3A//demo.dataverse.org&doi=doi:10.70122/MYT/ESTDOI'
 
 
+Help others getting started
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Its not always obvious to outsiders whether a Dataverse Dataset is a DataLad Dataset as well.
+To help others get started with your shared datasets, we recommend to add a short DataLad-specific description to your dataset's "Description" metadata.
+For convenience, a template to copy-and-paste can be found below.
+
+.. code-block:: text
+
+   This dataset is a <b>DataLad dataset</b> (https://www.datalad.org), published with
+   the <code>DataLad-dataverse</code> software (https://docs.datalad.org/projects/dataverse).
+   If you <code>"datalad clone"</code> it, it provides fine-grained data access down to the
+   level of individual files, and allows for tracking future updates. For this,
+   DataLad and datalad-dataverse are required. You can find installation instructions
+   at https://docs.datalad.org/projects/dataverse/settingup.html#installation.
+   Afterwards, you can clone it with the following command-line call (replace
+   the two placeholders <code>DATAVERSE-INSTANCE-URL</code> and <code>DATASET-DOI</code>): <br><br>
+
+       <code>datalad clone 'datalad-annex::?type=external&externaltype=dataverse&encryption=none&exporttree=no&url=DATAVERSE-INSTANCE-URL&doi=DATASET-DOI' my-dataset-clone</code> <br><br>
+
+   Once a dataset is cloned, it is a light-weight directory on your local
+   machine.
+   At this point, it contains only small metadata and information on the
+   identity of the files in the dataset, but not actual *content* of the
+   (sometimes large) data files.
+   After cloning a dataset, you can retrieve file contents by running
+   <code>"datalad get path/to/directory/or/file"</code>
+   This command will trigger a download of the files, directories, or
+   subdatasets you have specified.<br><br>
```

Comment:
```suggestion
   After cloning a dataset, you can retrieve file contents from available locations
   by running <code>"datalad get path/to/directory/or/file"</code>.
   This command will trigger a download of the files, directories, or
   subdatasets you have specified.<br><br>
```

Somewhat highlighting that it is not Dataverse that you'd retrieve the data from - although "available locations" is also very nondescriptive, so I don't insist.

Otherwise, just a missing full stop.